openwisp_notifications/templates/openwisp_notifications/email_template.html

```diff
@@ -0,0 +1,93 @@
+{% load i18n %}
+{% load l10n %}
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  {% block styles %}
+  <style type="text/css">
+    .background {
+      height: 100%;
+      background: linear-gradient(to bottom, #bb442c 50%, #464646 50%);
+      background-repeat: no-repeat;
+      background-attachment: fixed;
+      padding: 50px;
+    }
+
+    .container {
+      padding: 20px;
+      width: 75%;
+      margin: auto;
+      background-color: whitesmoke;
+      border-radius: 10px;
+    }
+
+    .notification-description {
+      text-align: center;
+      padding: 0px 30px;
+    }
+
+    .logo {
+      width: 100px;
+    }
+
+    table {
+      border-collapse: collapse;
+    }
+
+    td,
+    th {
+      padding: 10px;
+    }
+
+    .notification-header {
+      background-color: #464646;
+      color: white;
+    }
+
+    .notification-subject {
+      width: 100%;
+      text-align: right;
+    }
+
+    .notification-subject a {
+      color: white;
+    }
+  </style>
+  {% endblock styles %}
+</head>
+
+<body>
+  {% block body %}
+  <div class="background">
+    <div class="container">
+      <table style="width: 100%;">
+        <thead class="notification-header">
+          <tr>
+            <th><img src="{{logo}}" alt="Organization Logo" id="org-logo" class="logo"></th>
```

Comment:
it'd be nice if it was the organization logo (eg: each organization supplying their own logo), but it's not, so I would not use the words `org` and `Organization`